src/Nethermind/Ethereum.Blockchain.Block.Test/MetaTests.cs

```diff
@@ -33,7 +33,7 @@ public void All_categories_are_tested()
             string[] directories =
                 Directory.GetDirectories(AppDomain.CurrentDomain.BaseDirectory)
                 .Select(Path.GetFileName)
-                .Where(d =>d.StartsWith("bc") || d.StartsWith("vm"))
+                .Where(d =>d.StartsWith("bc"))
```

Comment:
vm tests are gone?

Reply:
Before changes the same VM tests were in Blockchain.Test, Blockchain.Block.Test and VM.Test (last one was not starting because of wrong directory). Now we have VM tests only in VM.Test